core/src/main/java/com/hamdan/forzenbook/core/UtilFunctions.kt

```
@@ -33,24 +36,38 @@ fun getBitmapFromUri(
     uri: Uri,
     context: Context,
     onError: () -> Unit,
-    onBitmapLoaded: (Bitmap) -> Unit
+    onBitmapLoaded: (Bitmap) -> Unit,
+    onImageSaved: (File?) -> Unit
```

Why nullable?

---

In case theres an issue creating the temporary file, though I suppose in that situation we just want to throw an error